keyring/backends/Windows.py

```diff
@@ -49,14 +257,19 @@ def value(self):
         """
         Attempt to decode the credential blob as UTF-16 then UTF-8.
         """
-        cred = self['CredentialBlob']
+
+        cred = self["CredentialBlob"]
+
+        # NOTE if we have our metadata, we know how to decode
+        if ATTRIBUTE_KEYWORD in self:
+            return cred.decode(self[ATTRIBUTE_KEYWORD]["encoding"])
+
         try:
-            return cred.decode('utf-16')
+            return cred.decode("utf-16")
```

Comment:
It would be helpful not to make capricious syntactical changes like this. The black config on this project allows for single-quote strings, so it would be easier to read the diff if it didn't have changes unrelated to the goal.

Reply:
Agreed.  I'm quite new at working on open source (although have been programming for nearly 50 years), and don't know a lot.  Is there a document for `keyring` giving guidance for a new developer?  If so happy to read it.  In any case, I'll revert to single-quote strings.

Reply:
No problem. Glad for your contribution and happy to help. There's no keyring-specific guidance, but as a general rule, and especially early on when working with a project, try to stick to changes that directly impact the issue at hand. See [this blog](https://blog.jaraco.com/how-to-write-perfect-pull-request/) for some guidance I wrote years ago, which is mostly still relevant.